src/victory-util/prop-types.js

```diff
@@ -138,6 +138,22 @@ export default {
   }),
 
   /**
+   * Check that the value is greater than zero.
+   */
+  greaterThanZero: makeChainable((props, propName, componentName) => {
+    const error = PropTypes.number(props, propName, componentName);
+    if (error) {
+      return error;
+    }
+    const value = props[propName];
+    if (value < 1) {
```

Comment:
If you make a validator called `greaterThanZero` it should fail if and _only if_ the value provided is less than or equal to zero. This method will fail for values between zero and one. I'm guessing you wrote this method in this way because for the `tickCount` validation you need an integer that is greater than zero.  We have an `allOfType` validator and an `integer` validator already written though, so you can do something like:
```
CustomPropTypes.allOfType([
  CustomPropTypes.integer, CustomPropTypes.greaterThanZero
])
```